docs/en/sql-reference/functions/array-functions.md

```diff
@@ -2159,7 +2162,17 @@ Alias: `arrayAUC`
 
 - `arr_scores` — scores prediction model gives.
 - `arr_labels` — labels of samples, usually 1 for positive sample and 0 for negative sample.
-- `scale` - Optional. Wether to return the normalized area. Default value: true. [Bool]
+- `scale` - Optional. Decides whether to return the normalized area. If false, returns the area under the (True Positives) TP x (False Positives) FP curve instead. Default value: true. [Bool]
```

Comment:
Guess it would be nice if you could document the valid data types for arguments `arr_scores`, `arr_labels`, `scale` and `arr_partial_offsets`. Admittedly, the argument types are not documented for most array functions ... but we need to start somewhere. See https://clickhouse.com/docs/en/sql-reference/functions/string-search-functions#position for an example what I mean.